docs/source/index.rst

```diff
@@ -36,6 +36,7 @@ modularity and versatility for a broad set of industry use cases.
    releases
    questions
    status
+   gobackextension
```

Comment:
Is this a UI label or invisible UI element ? If this is a visible label, then we need to have better name.

Reply:
No this is not visibile element. The purpose of this rst file is to create that link from original fabric documentation to fabric mod documentation.